doc/refman/Cases.tex

```diff
@@ -280,6 +280,19 @@
      end).
 \end{coq_example}
 
+The option {\tt Set Asymmetric Patterns} \optindex{Asymmetric Patterns}
+(off by default) removes the parameters from the pattern
```

Comment:
"removes parameters to constructors in patterns" would be more clear to me, with less need for all the preceding context